test/StaticValidatorTest.php

```diff
@@ -152,9 +152,38 @@ public function testPassingNullWhenSettingPluginManagerResetsPluginManager()
         $this->assertNotSame($plugins, StaticValidator::getPluginManager());
     }
 
-    public function testExecuteValidWithParameters()
+    public function parameterizedData()
     {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
-        $this->assertTrue(StaticValidator::execute(5, 'Between', ['min' => 1, 'max' => 10]));
+        return [
+            'valid-positive-range'   => [5, 'Between', ['min' => 1, 'max' => 10], 'assertTrue'],
+            'valid-negative-range'   => [-5, 'Between', ['min' => -10, 'max' => -1], 'assertTrue'],
+            'invalid-positive-range' => [-5, 'Between', ['min' => 1, 'max' => 10], 'assertFalse'],
+            'invalid-negative-range' => [5, 'Between', ['min' => -10, 'max' => -1], 'assertFalse'],
+        ];
+    }
+
+    /**
+     * @dataProvider parameterizedData
+     */
+    public function testExecuteValidWithParameters($value, $validator, $options, $assertion)
+    {
+        $this->$assertion(StaticValidator::execute($value, $validator, $options));
```

Comment:
In this case, it can be simplified even further: `assertSame($expected, $return)`, which is the approach I'm about to push.